ModAssistant/MainWindow.xaml.cs

```diff
@@ -267,6 +269,28 @@ private void OptionsButton_Click(object sender, RoutedEventArgs e)
             Properties.Settings.Default.Save();
         }
 
+        private void PlayButton_Click(object sender, RoutedEventArgs e)
+        {
+            if (Options.Instance.InstallType == "Steam")
+            {
+                Process.Start("steam://rungameid/620980");
+            }
+            else
+            {
+                string exePath = Path.Combine(Options.Instance.InstallDirectory, "Beat Saber.exe");
```

Comment:
Since this uses the .exe isn't it a bit redundant to use an If else? The Install Directory is already known right, so wouldn't we only use that?